scenarios/08_Outlaw_Base.cfg

```diff
@@ -29,12 +29,13 @@
         name=_"Gawen Hagarthen"
         type=Hagarthen
         profile=portraits/gawen.png
+        facing=ne
         team_name=good
         canrecruit=yes
         side=1
         recruit=Fencer,Peasant,Bowman,Spearman
         # Default gold when left unspecified is 100:
-        {GOLD 120 110 100}
+        {GOLD 124 112 100}
```

Comment:
Interesting change. :O

Reply:
basically just enough to get me an additional unit :-P